sql3/test/defs/defs_date_functions.go

```diff
@@ -241,5 +241,44 @@ var datePartTests = TableTest{
 			),
 			Compare: CompareExactUnordered,
 		},
+		{
```

Comment:
This test definition need to be organized by function. Earlier I mistakenly added toTimeStamp() tests to dataPart() tests. I am separating them as part of the dateTimeAdd() work. The broken down definitions would look like:

`var datePartTests = TableTest{......}

// toTimestamp tests
var toTimestampTests = TableTest{......}

// datetimeAdd tests
var datetimeAddTests = TableTest{......}

// datetimeName tests
var datetimeNameTests = TableTest{......}`

Reply:
Oh, good catch, I forgot to change the function being called on those. Not sure how that passed when I ran the tests.